source/guides/network/maintaining-a-symbol-node.rst

```diff
@@ -0,0 +1,125 @@
+
+Additionally, if you have enrolled in any of the :doc:`../../concepts/reward-programs` keeping up with server versions is a **mandatory requirement**.
+
+The main version to keep track of is the **Catapult Server version**, since this component is used by all :ref:`peer nodes <peer-node>`. Additionally, :ref:`API nodes <api-node>` need to keep track of the **REST component version** too.
+
+- **Finding out what are the latest versions**:
+
+  Whenever there is a new server version released it will be announced through the NEM |forum| and the |slack| channel. You can also keep an eye on the different |github| repositories and :doc:`the compatibility matrix page <../../compatibility>` to know the latest Server and REST versions.
+
+- **Finding out your node versions**:
+
+  You can find the versions used by your node in the `Symbol Explorer nodes list <http://explorer.symbolblockchain.io/nodes>`__.
+
+  Additionally, if your node is an :ref:`API node <api-node>`, you can also query it directly:
+
+  - **Server version**: ``/node/info`` endpoint, ``version`` property. `See an example <http://ngl-dual-104.symbolblockchain.io:3000/node/info>`__.
+
+    .. note::
+    
+       The ``version`` is encoded as a single 4-byte integer, where the Most-Significant Byte encodes the Major version, and the rest of bytes are the Minor, the Micro and the Patch version.
+
+       For example, version **16777472** (decimal) is **01000100h** (hex), which corresponds to version **1.0.1.0**.
+
+  - **REST version**: ``/node/server`` endpoint, ``restVersion`` property. `See an example <http://ngl-dual-104.symbolblockchain.io:3000/node/server>`__.
+
+    ``restVersion`` is a version string like **"2.3.6"**, for example.
+
+The exact procedure to update your node depends on how you created it. Read the appropriate section below.
+
+Update nodes created with Bootstrap
+===================================
+
+If you used Symbol Bootstrap to create your node (following the :doc:`running-a-symbol-node` guide), keeping it up to date is very simple.
+
+1. **Update Symbol Bootstrap** first (see the :doc:`using-symbol-bootstrap` guide for more details):
+
+   .. code-block:: bash
+
+      npm install -g symbol-bootstrap
+
+2. Then **move to the folder** where the node's data folder is (typically ``target``) and **make a backup copy** of it:
+
+   .. code-block:: bash
+
+      cp -r target target.BAK
+
+3. Finally, **stop and restart the node** with the ``--upgrade`` flag to update all necessary component versions.
+
+   The exact commands depend on whether your node was running in detached mode or not:
+
+   - **If you started in detached mode** with ``symbol-bootstrap start -d``:
+
+     From the folder containing the ``target`` folder type:
+
+     .. code-block:: symbol-bootstrap
+
+        symbol-bootstrap stop
+        symbol-bootstrap start --upgrade -d
```

Comment:
It would be safer if they provide the --customPreset customPresetFile.yml when doing the --upgrade

Reply:
Added!